hamming.js

```diff
@@ -6,12 +6,23 @@
 
 'use strict';
 
-function Hamming() {};
-
-Hamming.prototype.compute = function (strand1, strand2) {
-//
-// YOUR CODE GOES HERE
-//
+var Hamming = function(hamming){
+  this.hamming = hamming
 };
 
+Hamming.prototype.compute = function(strand1, strand2){
+  if(strand1 === strand2) {
+    return 0;
+  }
+  if (strand1.length !== strand2.length) {
+    throw new Error('DNA strands must be of equal length.');
+  }
+  var mutation_count = 0;
+  for (i = 0; i < strand1.length; i++){
+    if(strand1[i] !== strand2[i]){
+      mutation_count += 1;
+    }
+  }
+  return mutation_count;
+
```

Comment:
Wow, that looks good, but I think you just forget a final `};` to close your `function` definition :wink: !